src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -404,6 +410,79 @@ void FancyZones::MoveWindowIntoZone(HWND window, winrt::com_ptr<IZoneWindow> zon
     }
 }
 
+int RectWidth(const RECT& rect)
+{
+    return rect.right - rect.left;
+}
+
+int RectHeight(const RECT& rect)
+{
+    return rect.bottom - rect.top;
+}
+
+RECT FitOnScreen(HWND window, const RECT& windowRect, HMONITOR monitor, const RECT& monitorRect) 

```

Comment:
Unused parameter `monitor`

Reply:
Used it previously to get DPI from it, but now I've removed all DPI related calc. Will remove this one.

Reply:
Removed.